plugins/modules/one_vm.py

```diff
@@ -76,9 +78,9 @@
     type: bool
   wait:
     description:
-      - Wait for the instance to reach its desired state before returning. Keep in mind if you are waiting for instance to be in running state
-        it does not mean that you will be able to SSH on that machine only that boot process have started on that instance, see 'wait_for' example
-        for details.
+      - Wait for the instance to reach its desired state before returning. Keep in mind if you are waiting for instance to
+        be in running state it does not mean that you will be able to SSH on that machine only that boot process have started
+        on that instance. See the example using the M(ansible.builtin.wait_for) for details.
```

Comment:
Sorry, I missed one more thing:

Either add 'module', or remove 'the' before it:
```suggestion
        on that instance. See the example using the M(ansible.builtin.wait_for) module for details.
```
```suggestion
        on that instance. See the example using M(ansible.builtin.wait_for) for details.
```